### PR TITLE
feat: add min versions for mobile SDKs

### PR DESCRIPTION
**Pull request by Mercy811**
# Amplitude Developer Docs PR

## Description

https://amplitude.atlassian.net/browse/AMP-86273

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sdks/android-kotlin/index.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ The Kotlin Android SDK lets you send events to Amplitude. This library is open-s
 
 --8<-- "includes/size/java.md"
     selecting ["Amplitude Android Kotlin SDK"](https://mvnrepository.com/artifact/com.amplitude/analytics-android).
+
+!!! info "Minimum version"
+    Refer to [`build.gradle`](https://github.com/amplitude/Amplitude-Kotlin/blob/90ef21cdcadd69660b51d4f670d25a78835750e2/android/build.gradle#L24) for minimum API level required to use the package.
```
Comment (markzegarelli) [inline docs/data/sdks/android-kotlin/index.md:26]:
```suggestion
    For the minimum supported SDK version of this package, see [`build.gradle`](https://github.com/amplitude/Amplitude-Kotlin/blob/90ef21cdcadd69660b51d4f670d25a78835750e2/android/build.gradle#L24) on GitHub.
```
@Mercy811 The referenced code references "SDK" and "version". Is that correct, or should it be "API level"?

File: docs/data/sdks/android/index.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ This is the official documentation for the Amplitude Analytics Android SDK.
 --8<-- "includes/size/java.md"
     selecting "Android SDK".
 
+!!! info "Minimum version"
+    Refer to [`build.gradle`](https://github.com/amplitude/Amplitude-Android/blob/4e37e0f2d21c437af2b4f4bb58520b061a21a2be/build.gradle#L39) for minimum API level required to use the package.
```
Comment (markzegarelli) [inline docs/data/sdks/android/index.md:26]:
```suggestion
    For the minimum supported SDK version of this package, see [`build.gradle`](https://github.com/amplitude/Amplitude-Android/blob/4e37e0f2d21c437af2b4f4bb58520b061a21a2be/build.gradle#L39) on GitHub.
```
Same as above.

File: docs/data/sdks/ios-swift/index.md [hunk 3]
```diff
@@ -16,6 +16,9 @@ This is the official documentation for the Amplitude Analytics iOS SDK.
 --8<-- "includes/size/ios.md"
     `./measure_cocoapod_size.py --cocoapods AmplitudeSwift:0.7.0`.
 
+!!! info "Minimum versions"
+    Refer to [`Package.swift`](https://github.com/amplitude/Amplitude-Swift/blob/main/Package.swift#L8-L12) for minimum versions for platforms supported by the package
```
Comment (markzegarelli) [inline docs/data/sdks/ios-swift/index.md:20]:
```suggestion
    For the minimum supported platform versions of this package, see [`Package.swift`](https://github.com/amplitude/Amplitude-Swift/blob/main/Package.swift#L8-L12) on GitHub.
```

File: docs/data/sdks/ios/index.md [hunk 4]
```diff
@@ -25,6 +25,9 @@ This is the official documentation for the Amplitude Analytics iOS SDK.
 --8<-- "includes/size/ios.md"
     `./measure_cocoapod_size.py --cocoapods Amplitude:8.17.1`.
 
+!!! info "Minimum versions"
+    Refer to [`Package.swift`](https://github.com/amplitude/Amplitude-iOS/blob/dca2c744336e651ee285011282b86accc2ce3704/Package.swift#L8-L12) for minimum versions for platforms supported by the package.
```
Comment (markzegarelli) [inline docs/data/sdks/ios/index.md:29]:
```suggestion
    For the minimum supported platform versions of this package, see [`Package.swift`](https://github.com/amplitude/Amplitude-iOS/blob/dca2c744336e651ee285011282b86accc2ce3704/Package.swift#L8-L12) on GitHub.
```

File: docs/data/sdks/android-kotlin/index.md (hunk 1 above)
Comment (justin-fiedler) [inline docs/data/sdks/android-kotlin/index.md:26]: We should use the `main` branch (or equivalent) for the links

```suggestion
    Refer to [`build.gradle`](https://github.com/amplitude/Amplitude-Kotlin/blob/main/android/build.gradle#L24) for minimum API level required to use the package.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by justin-fiedler — APPROVED** [review]
Thanks @Mercy811. See comment to update links to use `main` branches. Otherwise looks great!

File: docs/data/sdks/android-kotlin/index.md (hunk 1 above)
Comment (Mercy811) [reply docs/data/sdks/android-kotlin/index.md:26]: It is correct. Not sure which word I should use here so copy from Apple dev docs [here](https://developer.apple.com/documentation/packagedescription/package/platforms)

**Review by markzegarelli — APPROVED** [review]

File: docs/data/sdks/android-kotlin/index.md (hunk 1 above)
Comment (Mercy811) [reply docs/data/sdks/android-kotlin/index.md]: updated

**Pull request merged**